src/pytom_tm/extract.py

```diff
@@ -69,6 +69,8 @@ def extract_particles(
         )
         cut_off = erfcinv((2 * n_false_positives) / search_space) * np.sqrt(2) * sigma
         logging.info(f'cut off for particle extraction: {cut_off}')
+    elif cut_off < 0:
+        cut_off = 0
```

Comment:
this should give a warning (as we might be silently overriding user settings here) or input sanitation has to happen in the arparse

Reply:
Added a logging.warning for this one